test/unit_tests/modules/util/expanded-state-spec.js

```diff
@@ -1,17 +1,19 @@
 'use strict';
 var chai = require( 'chai' );
-var sinon = require( 'sinon' );
 var expect = chai.expect;
+var sinon = require( 'sinon' );
+var es = require( '../../../../src/static/js/modules/util/expanded-state.js' );
 var jsdom = require( 'mocha-jsdom' );
+var $;
 
 describe( 'Event States', function() {
-  var $, es, sandbox, divExpanded, divClosed, openMenu;
+  var sandbox, divExpanded, divClosed, openMenu;
 
   jsdom();
 
+  $ = require( 'jquery' );
+
   before( function() {
-    $ = require( 'jquery' );
-    es = require( '../../../../src/static/js/modules/util/expanded-state.js' );
```

Comment:
Yeah, sorry discovered this in https://github.com/cfpb/cfgov-refresh/pull/1007#discussion_r40253020 Pull down the latest.